src/Uno.Templates/content/unoapp/.template.config/template.json

```diff
@@ -422,9 +422,28 @@
     },
     "http": {
```

Comment:
The tests in **action.yml** will need to change since now they consider `http` to be boolean.

https://github.com/unoplatform/uno.templates/blob/2bce19094a415d46ca8875e602824d5c9dc7e8ee/.github/actions/ci/generate-test-matrix/action.yml#L33